### ISSUE TITLE
swagger url incorrect

### ISSUE THREAD

**Issue by Mercbot7** [rikatz/kubepug#196]
While trying to check deprecations I am receiving error:

```shell
Error: could not download the swagger file https://raw.githubusercontent.com/kubernetes/kubernetes/1.20.11/api/openapi-spec/swagger.json
time="2021-09-24T15:46:54-04:00" level=error msg="An error has ocurred: could not download the swagger file https://raw.githubusercontent.com/kubernetes/kubernetes/1.20.11/api/openapi-spec/swagger.json"
```
The issue here is that the url is incorrect.

The correct path is:

```shell
https://raw.githubusercontent.com/kubernetes/kubernetes/v1.20.11/api/openapi-spec/swagger.json
```
The `v` before the version is missing

in the [downloader.go](https://github.com/rikatz/kubepug/blob/main/pkg/utils/downloader.go) utility it shows:

https://github.com/rikatz/kubepug/blob/9f4a2a2737c7d4755600a9bd2defb76180e5dbf7/pkg/utils/downloader.go#L13-L14

and these are used later:
https://github.com/rikatz/kubepug/blob/9f4a2a2737c7d4755600a9bd2defb76180e5dbf7/pkg/utils/downloader.go#L62

Not sure when it was changed but the current urls all have the `v` before the version number.

Workaround:
I have used the following to get around this, basically manually downloading and naming per the convention used in the downloader.go utility.
https://github.com/rikatz/kubepug/blob/9f4a2a2737c7d4755600a9bd2defb76180e5dbf7/pkg/utils/downloader.go#L57

basically it is `swagger-<k8s version>.json`

```shell
k_swag_version="1.20.11"
mkdir ~/kube_api_swagger_files
curl https://raw.githubusercontent.com/kubernetes/kubernetes/v${k_swag_version}/api/openapi-spec/swagger.json --output ~/kube_api_swagger_files/swagger-${k_swag_version}.json

kubectl deprecations --k8s-version $k_swag_version --swagger-dir ~/kube_api_swagger_files --input-file kubernetes.yaml

```

**Comment by rikatz** [comment]
Hum, maybe they have changed this.

Let me take a look. If you can also provide a PR, would be more than welcome :D

**Comment by Morishiri** [comment]
It says in the [readme](https://github.com/rikatz/kubepug#checking-a-kubernetes-cluster) to use the version with `v`.

It works for me when used like this:
```
kubepug --k8s-version=v1.22.2
```

**Comment by Mercbot7** [comment]
@Morishiri @rikatz Sorry for making an issue out of nothing. If i had RTFM this would not even exist. Sorry about that. Iam closing this issue and apologize for wasting your time.

**Comment by rikatz** [comment]
No need to be sorry :) thanks for using the project :)
